<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Анализ отзывов</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>921</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="dataLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Выбор данных</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="urlLabel">
        <property name="text">
         <string>Название книги на английском языке </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="urlEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="title">
       <string>Выбор анализатора</string>
      </property>
      <widget class="QRadioButton" name="allRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>917</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Все</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="watsonRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>917</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>IBM Watson</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="monkeyRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>917</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>MonkeyLearn</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="analyseButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>150</horstretch>
        <verstretch>8</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Запустить анализ</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>10</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Результаты анализа</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
